os/binfmt/libelf/libelf_addrenv.c

```diff
@@ -77,6 +77,37 @@
 /****************************************************************************
  * Private Functions
  ****************************************************************************/
+#ifdef CONFIG_OPTIMIZE_APP_RELOAD_TIME
+static void allocateregions(FAR struct elf_loadinfo_s *loadinfo)
+{
+	size_t sizes[3] = {loadinfo->textsize, loadinfo->rosize, loadinfo->binp->ramsize};
```

Comment:
Can you use a macro instead of using "3" everywhere?

Reply:
Done.

Reply:
I still see the "3" in code, change size_t sizes[3], uintptr_t *allocs[3] also

Reply:
Fixed.